src/test/groovy/com/palantir/gradle/revapi/RevapiSpec.groovy

```diff
@@ -421,6 +421,142 @@ class RevapiSpec extends IntegrationSpec {
         runTasksSuccessfully("tasks")
     }
 
+    def 'breaks detected in conjure projects should be limited to those which break java but are not caught by conjure-backcompat'() {
+        when:
+        buildFile << """
+            buildscript {
+                repositories {
+                    maven { url 'https://dl.bintray.com/palantir/releases/' }
+                    mavenCentral()
+                }
+            
+                dependencies {
+                    classpath 'com.palantir.gradle.conjure:gradle-conjure:4.13.3'
+                }
+            }
+                        
+            allprojects {
+                group = 'revapi.test'
+                version = '1.0.0'
+                
+                repositories {
+                    maven { url 'https://dl.bintray.com/palantir/releases/' }
+                    mavenCentral()
+                }
+            }
+        """
+
+        def apiDir = addSubproject 'api', """
+            apply plugin: 'com.palantir.conjure'
+            
+            dependencies {
+                conjureCompiler 'com.palantir.conjure:conjure:4.6.2'
+                conjureJava 'com.palantir.conjure.java:conjure-java:4.5.0'
+            }
+            
+            subprojects {
+                apply plugin: '${TestConstants.PLUGIN_NAME}'
+
+                revapi {
+                    oldVersion = project.version
+                }
+
+                dependencies {
+                    compile 'com.palantir.conjure.java:conjure-lib:4.5.0'
+                    compile 'com.palantir.conjure.java:conjure-undertow-lib:4.5.0'
+                    compile 'com.squareup.retrofit2:retrofit:2.6.2'
+                }
+                
+                apply plugin: 'maven-publish'
+
+                ${mavenRepoGradle()}
+                ${testMavenPublication()}
+            }
+        """
+
+
+        new File(apiDir, 'api-objects').mkdirs()
+        new File(apiDir, 'api-jersey').mkdirs()
+        new File(apiDir, 'api-retrofit').mkdirs()
+        new File(apiDir, 'api-undertow').mkdirs()
+        settingsFile << """
+            include 'api:api-objects'
+            include 'api:api-jersey'
+            include 'api:api-retrofit'
+            include 'api:api-undertow'
+        """
```

Comment:
isn't there are nice `addSubproject` method for this?

Reply:
That adds `build.gradle` which most conjure projects don't have - I guess it's fine to have an empty one though

Reply:
Add, it can't add sub-sub projects, but I guess there's no reason I couldn't apply conjure to the root project